sample/legacy/format_list/format_list_sample.json

```diff
@@ -0,0 +1,11 @@
+// Target-Table: ESS_MS_Controls_FormatList
+{
+  "internalName": "<internal-name>",
+  "description": "<description>",
+  "items": [
+    {
+      "name": "<entry-name>",
```

Comment:
is this needed? its not in the ess_ms_controls_formatlist table

Reply:
`name` --> `displayName`